cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -75,7 +75,7 @@
         (size OperandSize)
         (rd WritableReg)
         (rn Reg))
-
+      
```

Comment:
As I mentioned in our meeting, you'll want to edit your branch such that these non-significant changes don't appear in the PR. Thanks!